### ISSUE TITLE
[BUG] Range: unable to add accessible name to kirby-range

### ISSUE THREAD

**Issue by Fuzzy3** [kirbydesign/designsystem#3805]
- [X] I have written a [good issue](https://github.com/kirbydesign/designsystem/wiki/The-Good%3A-Issue)

## Describe the bug

There is an accessibility issue in the Kirby Range component where integrators are unable to add an accessible name. The name needs to be placed somewhere in the Shadow DOM. This cannot be fixed “outside” Kirby, therefore changes has to be made within the code base.

## Describe how to reproduce the bug
1. Use Kirby Range component
2. Use LevelAccess tool
3. See the error present itself through the tool:
4. Error: “Element with role ‘slider’ missing an accessible name”
 
## What was the expected behavior?
Expected slider component to be accessible out-of-the-box.

## Please complete the following information:
- OS: MacOS
- Browser Chrome

<hr />

## Checklist:

The following tasks should be carried out in sequence in order to follow [the process of contributing](https://github.com/kirbydesign/designsystem/blob/develop/.github/CONTRIBUTING.md#the-process-of-contributing) correctly.

### Verification
To make sure the bug is not intended behaviour; it should be verified by a member of team Kirby before moving on to implementation. 

- [ ] Make sure the `NOT verified` label has been removed by a member of [team Kirby](https://github.com/kirbydesign/designsystem/blob/develop/.github/SUPPORT.md#team-kirby); do not proceed until this is done.

### Implementation 
The contributor who wants to implement this issue should: 

- [ ] Make sure you have read: "[Before you get coding](https://github.com/kirbydesign/designsystem/blob/develop/.github/CONTRIBUTING.md#before-you-get-coding)".
- [ ] Signal to others you are working on the issue by assigning yourself.
- [ ] Create a branch from the [develop branch](https://github.com/kirbydesign/designsystem/tree/develop) following our [branch naming convention](https://github.com/kirbydesign/designsystem/wiki/The-Good%3A-Branch). 
- [ ] Create a test that reproduces the bug following guidelines in: "[The good: Test](https://github.com/kirbydesign/designsystem/wiki/The-Good%3A-Test)". 
- [ ] Write code to fix the bug

### Review
Once the issue has been implemented and is ready for review:

- [ ] Do a [self-review](https://github.com/kirbydesign/designsystem/wiki/The-Good%3A-Self-review). 
- [ ] Create a pull-request.
